src/input/input.c

```diff
@@ -7,7 +7,7 @@ void
 input_handlers_init(struct seat *seat)
 {
 	cursor_init(seat);
-	keyboard_init(seat);
+	keyboard_init(seat, false);
```

Comment:
```suggestion
	keyboard_init(seat, /* reconfig */ false);
```

Reply:
Duplicating an entire function except for the first two lines is much worse than adding an argument, and the comment is in keeping with the style of other calls that annotate the meaning of non-obvious arguments like this. Also note that my original suggestion was to rename this to `keyboard_configure` so there would be less confusion about repeated calls.

However, as Consolatis pointed out in IRC, you can probably drop the extra argument altogether and check in `keyboard_init` whether `seat->keyboard_group == NULL` to determine whether you need to perform an initial creation or just reconfigure.

Reply:
> Duplicating an entire function except for the first two lines is much worse than adding an argument...

Why? In what way is it "much worse"? It's a tiny function - barely ten lines. As a fraction of the size of the thousands of lines of labwc code, the effect of duplicating it is negligible. (Some compilers would even optimise that out anyway.) And calling it "keyboard_configure" doesn't make it any more obvious as to what the argument means when it is called.

My preference would be to retain separate keyboard_init and keyboard_reconfig function calls with, if repeating code is felt to be unacceptable, the common code removed to a third function which they both call. That makes it very obvious what is going on to anyone reading the code with no need for comments at all.

Reply:
Copy-pasting functions leads to divergence and latent bugs when somebody decides to modify the behavior of one and forgets to synchronize the other. The line count is irrelevant; any time you are trying to do the same non-trivial task more than once, that task should be wrapped in a function.

However, as I noted before, there is a better way to do this without the need for any change in the call signature.